client-service-worker/src/__dumber-gist-worker.js

```diff
@@ -132,7 +134,7 @@ addEventListener('fetch', e => {
       return fetch(e.request).then(response => {
         if (!response.ok && pathname.length > 1) {
           // Try font/image from jsdelivr
-          return fetch(`//cdn.jsdelivr.net/npm${pathname}`, {mode: 'cors'}).then(response => {
+          return fetch(`${JSDELIVR_CDN_URL}/npm${pathname}`, {mode: 'cors'}).then(response => {
```

Comment:
This line is the only reason for the new gulp task in service worker. I will seek to simplify the setup after merging this PR.